.vib/joomla/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+files:
+  - paths:
+      - /opt/bitnami/apache/modules/libphp.so
```

Comment:
This is nice, given that we do not have a libphp container. As such we need to test libphp-related logic in those containers that include said components. That said, there is already a common goss template on the works called [check-apache-libphp.yaml](https://github.com/bitnami/containers/pull/29156/files#diff-2a8ebe41b0c46f27150d4084b037555c1b5700c4488459f7e0c7514e8671204f) 😁 That should simplify these repetitive checks.

Once @migruiz4 merges the related PR I think you can add it to your `goss.yaml` file.